docs/operation_center/actions.md

```diff
@@ -0,0 +1,170 @@
+
+##### Create module 
+With the Create module a user will be able to `Create`:
+
+- A new incident
+- An alert
+- A new asset
+
+##### Update module 
+With the Update module a user will be able to `Update`:
+
+- An incident
+
+##### Patch module 
+With the Patch module a user will be able to `Patch`:
+
+- An alert
+
+##### Add module 
+With the Add module a user will be able to `Add`:
+
+- Owners to an asset
+- Key to the asset
+- Attribute to the asset
+
+##### Post module 
+With the Post module a user will be able to `Post`:
+
+- An asset
+- An owner from an asset
+- A key form an asset
+- Attribute to the asset
+
+##### Delete module 
+With the Delete module a user will be able to `Delete`:
+
+- A comment on an alert
+
+##### Trigger an action on the event workflow
+Is is possible to trigger an action on an alert as explained in the webhook section. For instance the following image shows how to properly close an alert afer analysing it.
+
+![SEKOIA.IO Operation Center Trigger an alert on worksflow](/assets/trigger_an_action_on_the_alert_workflow.png){: style="max-width:600px"}
+
+As of today, the available actions_uuid are the following :
+```r
+Action:"Acknowledge", 	Description: "Acknowledge the alert", 	action_uuid:"937bdabf-6a08-434b-b6d3-d7447e4e452a"
+Action:"Validate", 		Description: "Validate the alert", 		action_uuid:"c39a0a95-aa2c-4d0d-8d2e-d3decf426eea"
+Action:"Reject", 		Description: "Reject the alert", 		action_uuid:"ade85d7b-7507-4026-bfc6-cc006d10ddac"
+Action:"Close", 		Description: "Close the alert", 		action_uuid:"1390be4e-ced8-4dd6-9bed-573471b235ab"
+```
+
+Other Sekoia modules are available, such as:
+
+- Associate new rule on a case
+- Associate new alert on a case
+- Activate or deny a countermeasure
+
+### External Data Enrichers
+
+Most of the time an analyst uses external tools in order to better understand the severity of the alert.
+> If you have subscribed to one or more of the following online tools, you should be able to get an API key, an simply use it on your playbooks.
+
+##### IKnowWhatYouDownload
+The module allows to request information such as: 
+
+- IP history
+- IP Check
+- IP List
+
+##### Onyphe
+The module allows to request information such as: 
+
+- Scan (SYN, Onion...)
+- Lookups (Md5, reverse DNS)
+- Get IP additional information (Threat list, geolocalisation...)
+- Scan Domain Name CTLs
+
+##### RiskIQ
+The module allows to request information such as: 
+ 
+- Whois
+- Reverse Whois (organisation, nameserver, name, email, address)
+- Passive DNS (Name, IP, Hex)
+- SSL host, certificate (SHA-1, Serial Number) / DNS
+
+##### Showdan
+The module allows to request information such as: 
+
+- Search for Host information
+- Get DNS reverse, resolve, domain
+
+##### Virustotal
+The module allows to request information such as: 
+
+- Scan (URL, IP, Hash, File, Domain)
+
+### Notifications
+
+##### PagerDuty
+This module allow the analysts to receive notification in PagerDuty when a new alert is raised.

```

Comment:
Why when a new alert is raised ? Everythink can't be notified ?

Reply:
je crois qu'il est buggé sur Chrome. Je n'arrive pas à l'utiliser, ni afficher de variables d'output.